xbmc/filesystem/IDirectory.cpp

```diff
@@ -75,9 +75,9 @@ bool IDirectory::IsAllowed(const CURL& url) const
     if (folder.size() <= 3) // cannot be a vcd variant
       return true;
 
-    if (!StringUtils::CompareNoCase(folder, "vcd") &&
-        !StringUtils::CompareNoCase(folder, "MPEGAV") &&
-        !StringUtils::CompareNoCase(folder, "CDDA"))
+    if (StringUtils::EqualsNoCase(folder, "vcd") ||
```

Comment:
the equivalent would strictly speaking be StartsWithNoCase, but i do not think that was the original intention in any case so this should be fine.